### PR TITLE
Add Movelinks2.app v1.3.25

### PR DESCRIPTION
**Pull request by bdossantos**
Moveslink is a software that connects your Suunto device to Movescount.

http://www.movescount.com/connect/moveslinkmac/Suunto_Ambit

### CODE REVIEW

**Comment by victorpopkov** [comment]
Thank you for the contribution. The download URL also needs a comment `# cloudfront.net is the official download host per the vendor homepage`. Furthermore, you can use string interpolation in `url` to include the version number `#{version.dots_to_underscores}`. In addition, the HTTPS for download URL is available which is more preferred.

Moreover, there is an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md) available for this app: https://d1c229iib3zm7m.cloudfront.net/mac/appcast.xml.

Most likely this cask also needs `auto_updates true`.
```ruby
  # cloudfront.net is the official download host per the vendor homepage
  url "https://d1c229iib3zm7m.cloudfront.net/mac/Moveslink2_#{version.dots_to_underscores}.dmg"
  appcast 'https://d1c229iib3zm7m.cloudfront.net/mac/appcast.xml',
          checkpoint: 'b628c933441bef876271f84cf5cc14d229a3b4b7c0ce053cefcf5dbe338effcf'
```

**Comment by vitorgalvao** [comment]
Small correction, comment needs to be `# d1c229iib3zm7m.cloudfront.net was verified as official when first introduced to the cask`.

Also, could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

File: Casks/movelinks.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+          checkpoint: 'b628c933441bef876271f84cf5cc14d229a3b4b7c0ce053cefcf5dbe338effcf'
+  name 'Movelinks'
+  homepage 'http://www.movescount.com/connect/moveslinkmac/Suunto_Ambit'
+  license :unknown # TODO: change license and remove this comment; ':unknown' is a machine-generated placeholder
+
+  auto_updates true
+
+  app 'Moveslink2.app'
+
+  uninstall delete:     '/Applications/Movelinks2.app',
```
Comment (vitorgalvao) [inline Casks/movelinks.rb]: This `delete:` needs to go. We do it automatically.

**Comment by bdossantos** [comment]
Done + squashed

**Comment by victorpopkov** [comment]
Thank you for the contribution. It was merged directly as 34c5529ef704c45891a36fe7d7aa658275ec3218 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**